Formula/wxpython.rb

```diff
@@ -18,11 +18,17 @@ class Wxpython < Formula
   end
 
   depends_on "freetype"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "numpy"
   depends_on "pillow"
+  # The current stable release 4.1.1 is incompatible with Python 3.10.
+  # Patching will require multiple commits to fix (e.g. Fedora uses ~10).
+  # TODO: Update to `python@3.10` on next release.
+  # Ref: https://bugs.archlinux.org/task/73763?project=5&string=python-wxpython
+  # Ref: https://src.fedoraproject.org/rpms/python-wxpython4/commits/
+  # Ref: https://github.com/wxWidgets/Phoenix/issues/2038
   depends_on "python@3.9"
```

Comment:
Testing locally, I think newer `setuptools`/`pip` has impacted build. May need to figure out exact issue and see if there are any upstream patches to backport.